src/naresh/Frequency.java

```diff
@@ -0,0 +1,25 @@
+
+package naresh;
+
+public class Frequency {
+
+	void display(String name) {
+
+		int count = 0;
+		for (int index = 0; index < name.length(); index++) {
+
+			if (name.charAt(index) == 't') {
```

Comment:
try to create generic methods, means user can give you one character and you have to print its frequency, so you will get character from parameters. Your code is only working for character 't', not for other characters.